recipes/crow/all/conanfile.py

```diff
@@ -0,0 +1,46 @@
+from conans import ConanFile, tools, CMake
+import os
+
+required_conan_version = ">=1.28.0"
+
+class CrowConan(ConanFile):
+    name = "crow"
+    homepage = "https://github.com/ipkn/crow"
+    description = "Crow is C++ microframework for web. (inspired by Python Flask)"
+    topics = ("conan", "web", "microframework", "header-only")
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "arch", "build_type"
+    generators = "cmake", "cmake_find_package", "cmake_find_package_multi"
+    exports_sources = ["patches/*"]
+    license = "BSD3"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        self.requires("openssl/1.1.1h")
+        self.requires("boost/1.69.0")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "crow-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+        tools.patch(**self.conan_data["patches"][self.version])
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.configure(source_folder=self._source_subfolder)
+        return cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
```

Comment:
Is this required? it's header only

Reply:
The `_configure_cmake` is only used here if the answer is yes

Reply:
Yes, it creates an "amalgamate" header only file during the CMake process, this it is required.

Reply:
so remove no_copy_source, because as it suggests, won't copy sources files to build folder

Reply:
Please add a comment, so we dont forget :smiley: